lib/client-api/src/story_store.ts

```diff
@@ -308,6 +303,17 @@ export default class StoryStore extends EventEmitter {
     this.pushToManager();
   }
 
+  setStoryState(id: string, newState: StoryState) {
+    if (!this._data[id]) throw new Error(`No story for id ${id}`);
+    const { state } = this._data[id];
+    this._data[id].state = { ...state, ...newState };
```

Comment:
Is there any way to delete something from state? Do we care?

Reply:
Not currently. Let's wait until that use case comes up, maybe?